tests/test_BoardConverter.py

```diff
@@ -0,0 +1,23 @@
+import unittest
+
+from board import Board
+from board_converter import BoardConverter
+from point import Point
+
+
+class BoardConverterTestCase(unittest.TestCase):
+    def test_convert_to_string(self):
+        board = Board(3, 4)
+        points = [Point(1, 2), Point(1, 3), Point(2, 0), Point(2, 1), Point(2, 2), Point(2, 3)]
```

Comment:
perhaps it is worth making a row from `Point`?

point : .. / 
point: ../
...

Reply:
Tnx for proposition